src/panels/config/voice-assistants/dialog-voice-assistant-pipeline-detail.ts

```diff
@@ -253,7 +267,7 @@ export class DialogVoiceAssistantPipelineDetail extends LitElement {
         name: data.name!,
         language: data.language!,
         conversation_engine: data.conversation_engine!,
-        conversation_language: data.conversation_language ?? null,
+        conversation_language: this._getDefaultConversationLanguage(),
```

Comment:
This is wrong I would say, we will always overrule the info in `data` now, with the first supported language?

Reply:
We should only do this if `data.conversation_language` is not in `_supportedConversationLanguages`?